server/src/simple-signal-server.js

```diff
@@ -59,7 +59,10 @@ SimpleSignalServer.prototype._onDiscover = function (socket, metadata) {
       })
     }
   })
-  self.peers.push(socket.id)
+
+  if (!self.peers.include(socket.id)) {
```

Comment:
Not sure if I should use include... I used it since it will run on Node.js. Maybe is a good idea to define [engines](https://docs.npmjs.com/files/package.json#engines) or [enginesStrict](https://docs.npmjs.com/files/package.json#enginestrict) on the package.json of the server?

Reply:
Okay, build failed with this change, I'll find the reason later... For the moment I made another commit replacing `Array.prototype.includes()` for  `Array.prototype.indexOf()`.